source/_posts/Citycross.md

```diff
@@ -0,0 +1,29 @@
+
+Občas se mi zachce exotiky. To pak do kapsy přihodím jízdenku a nechám se odvézt metrem nebo tramvají někam daleko. Třeba na tu periferii, kam se jinak jezdí jen z donucení. Nechám se vyplivnout na konečné a s pomocí podvědomé buzoly trasuju cestu domů. Napříč hlavními tahy. Když uzřu lákavou odbočku, tak do ní zahnu. Zásadně se nevracím stejnou cestou zpět ač to občas stojí botu zabořenou v bahně, trní zapíchané ve stehnech nebo bundu prodřenou o plot, jak se přes něj sápu. A když už spojujeme běh a MHD, zkoušeli jste někdy vyběhnout eskalátory na Náměstí Míru v protisměru?
+
+Ploty a zdi bývají problém čím dál tím častěji. Někdo možná ještě pamatuje průchody a vnitrobloky běžně otevřené. Já možná budu vyprávět dětem, jak se kdysi stavěla bydlení, která se neshovávala za plot s bránou a recepcí. Prostupnost je půl klíče k městu, kde běh dělá radost. Málo co potěší tak, jako na zdánlivě slepém konci průmyslového areálu najít díru v plotě a za ní vyšlapanou pěšinu. Jestli se někdy radikalizuju, tak v tílku a s nůžkama v ruce. Budu prostřihávat (staro)nové průchody skrz ploty, které stojí akorát všem nazmar. Snad mi to kdyžtak rozmluví kamarádi.
+
+A těch je ve městě vždy dostatek. Byla by hloupost ignorovat, že ve městě se městná spousta lidí a nevzít nějaké z nich s sebou na výběh. Citycross jako drzý a spontánní řez uličním prostorem není udržitelný s rostoucí velikostí skupiny. Někde mezi dvěma a třema běžci leží optimum, ve kterém se jejich individuální kreativita a spontánnost doplňují. Můj oblíbený režim spočívá v druhém běžci zezadu dávajícím povely tomu prvnímu, jakým směrem se má vydat. Druhý se může soustředit na vymýšlení trasy, zatímco slepě kopíruje kroky prvního, který má volné nohy na prošlapávání cesty.
+
+Jako běžec můžu ~~jít~~běžet městu naproti volbou bot, kreativní trasou, odvahou vkročit do pochybných zákoutí, vycestováním za příjemnější lokalitou, sluchátky, které odstíní hluk (dalším hlukem). Mohlo a mělo by nás ale taky zajímat, jak může jít město naproti nám, běžcům. Jaké objektivní kvality dělají město běhatelnější? Zmíněná prostupnost je důležitá. Obdobnou váhu pro mě má lidské měřítko, tak jak o něm [píše](https://www.goodreads.com/book/show/8556291-cities-for-people) Jan Gehl.
+
+Měřítko běhu zapadá někam mezi chůzi a jízdu na kole. Z chůze si bere hmatové vnímání okolí chodidly a prakticky neomezené možnosti terénu. Z kola zase vyšší rychlost a z ní plynoucí setrvačnost a neochotu zastavovat. Na škále měřítek různých druhů pohybu jsem jako běžec téměř na spodu a přirozeně vyhledávám ty části města, které mi jsou měřítkem blízké.
```

Comment:
`Na škále měřítek různých druhů pohybu jsem jako běžec téměř na spodu a přirozeně vyhledávám ty části města, které mi jsou měřítkem blízké.` tady je to možná lehce kostrbatý. Chvíli mi trvalo pochopit co si tím myslel - jakože si jako běžec víc v "nebezpečí" v centru a tak radši budeš běhat někde na okraji, resp. v místech kde není takový traffic?